modules/apps/foundation/user-associated-data/user-associated-data-web/src/main/java/com/liferay/user/associated/data/web/internal/portlet/action/ManageUserAssociatedDataEntityTypesMVCRenderCommand.java

```diff
@@ -64,6 +65,29 @@ public String render(
 		return "/manage_user_associated_data_entity_types.jsp";
 	}
 
+	private List<UADEntityTypeComposite> _getUADEntityTypeComposites(
```

Comment:
Since the `UADEntityTypeComposite` class is strictly for display logic and not an API to be implemented by the other `uad` modules, we should move it from the `api` module to the `web` module.